src/components/apps/dungeon/footer.tsx

```diff
@@ -44,6 +44,12 @@ import { get_discord_messages, DiscordMessage } from './utils';
 //  dungeon constants
 import { DUNGEON_FONT_SIZE} from './constants';
 
+interface DefeatedEnemyInterface {
+    characterEmoji: string | undefined;
+    enemyEmoji: any;
```

Comment:
can this by string | undefined rather than any?

Reply:
Changing and commiting the changes